sdk/containerregistry/container-registry/src/containerRepositoryClient.ts

```diff
@@ -345,11 +345,22 @@ export class ContainerRepositoryClient {
     });
 
     try {
-      return await this.client.containerRegistryRepository.updateManifestAttributes(
+      const properties = await this.client.containerRegistryRepository.updateManifestAttributes(
         this.repository,
         digest,
         updatedOptions
       );
+      return {
+        ...properties,
+        writeableProperties: properties.writeableProperties
+          ? {
+              canDelete: properties.writeableProperties.canDelete,
+              canList: properties.writeableProperties.canList,
+              canRead: properties.writeableProperties.canRead,
+              canWrite: properties.writeableProperties.canWrite
+            }
+          : undefined
```

Comment:
Why? Does `properties.writeableProperties` have more props?

Reply:
Yes currently the service returns more than we want to expose.